src/twig.expression.js

```diff
@@ -596,11 +596,39 @@ module.exports = function (Twig) {
 
                 output.push(token);
             },
-            parse(token, stack) {
+            parse(token, stack, context) {
                 const input = stack.pop();
-                const {params} = token;
+                let {params} = token;
                 const state = this;
 
+                if (parseInt(params[0], 10).toString() === params[0]) {
+                    params[0] = parseInt(params[0], 10);
+                } else {
+                    const value = context[params[0]];
```

Comment:
This only allows very simple variable names. Technically PHP Twig supports full expressions as slice parameters, like accessing a member value of a map or a math expression. However, I'm just mention this to point it out. I'm fine with allowing this simple case to push things forward and don't thing it should be a blocker to get this merged.

Reply:
Thanks for pointing this out!
I wasn't aware of it.

For now I think it is OK to allow these simple cases; I'll create an issue in order to support full expressions; but have to look into the current implementation in PHP before doing that.